src/Amazon.ElasticBeanstalk.Tools/EBUtilities.cs

```diff
@@ -114,20 +115,21 @@ public static void SetupPackageForLinux(IToolLogger logger, EBBaseCommand comman
             // Setup Procfile
             var procfilePath = Path.Combine(publishLocation, "Procfile");
 
-            if(File.Exists(procfilePath))
+            if (File.Exists(procfilePath))
             {
                 logger?.WriteLine("Found existing Procfile file found and using that for deployment");
                 return;
             }
 
             logger?.WriteLine("Writing Procfile for deployment bundle");
+            var projectLocation = Utilities.DetermineProjectLocation(command.WorkingDirectory,
```

Comment:
The caller of this method has already used `DetermineProjectLocation` to determine the location. Update this method signature to pass in the project location so we don't need to detect the location again. Since this is a tool not a library we can change the public signature.